apps/examples/ble_rmc/ble_rmc_main.c

```diff
@@ -447,15 +460,16 @@ int ble_rmc_main(int argc, char *argv[])
 
 		if (argc == 4 && strncmp(argv[2], "del", 4) == 0) {
 			int cnt = 0;
-			uint8_t mac[BLE_BD_ADDR_MAX_LEN] = { 0, };
+			ble_addr addr[1] = { 0, };
```

Comment:
it's good idea to make char[] to struct. it's easy to read.

Reply:
Thanks a lot